dot/network/service.go

```diff
@@ -267,6 +269,7 @@ func (s *Service) Start() error {
 	go s.logPeerCount()
 	go s.publishNetworkTelemetry(s.closeCh)
 	go s.sentBlockIntervalTelemetry()
+	s.streamManager.start()
```

Comment:
any reason why this spawns it's own goroutine instead of calling it via `go s.streamManager.start()`

Reply:
I see that there's a `ctx` created on `NewService`.  Do we need both the `closeCh` and `ctx`?

Reply:
no particular reason, I like it like this since the main network service doesn't need to be concerned with what the streamManager does